packages/kbn-reporting/public/share/share_context_menu/register_pdf_png_modal_reporting.tsx

```diff
@@ -111,14 +111,6 @@ export const reportingScreenshotShareProvider = ({
       objectType,
       sharingData,
     };
-    const isJobV2Params = ({
-      sharingData: _sharingData,
-    }: {
-      sharingData: Record<string, unknown>;
-    }) => _sharingData.locatorParams != null;
-
-    const isV2Job = isJobV2Params(jobProviderOptions);
-    const requiresSavedState = !isV2Job;
```

Comment:
The V2 PDF/PNG implementations do not require saved state, meaning a user can create a report of a dashboard without having to save it first.